packages/stryker-karma-runner/src/KarmaTestRunner.ts

```diff
@@ -77,6 +79,10 @@ export default class KarmaTestRunner implements TestRunner {
     this.server.start();
   }
 
+  private readConfig(options: RunnerOptions): ConfigOptions {
```

Comment:
Can't we pass `options.strykerOptions` to this method? Or would it be cleaner to just pass the options? Else the caller has to know what `readConfig` will do.

Reply:
It hides the implementation detail. Let's leave it for now.